### ISSUE TITLE
multi-scale model performance compared to single scale model

### ISSUE THREAD

**Issue by greathope** [layumi/AICIty-reID-2020#5]
I want to know  how much performance has been improved under same circumstances.

**Comment by layumi** [comment]
Multi-scale could generally improve the performance about 0.5%

**Comment by greathope** [comment]
> Do you mean the multi-scale inputs?
> Multi-scale inputs could generally improve the performance about 0.5%

hi, I mean how much performance can sophisticated re-id network architecture boost compared to the vanilla re-id baseline, I have not found comparision in your paper~

**Comment by layumi** [comment]
Hi @greathope 
Sorry. It is due to the competition timeline, which asks that the paper should be completed in 5 days. 
Therefore, we do not have enough time to conduct comprehensive ablation studies. 

As far as I remember, the pytorch baseline model on the validation set is around 38~40% mAP.

**Comment by greathope** [comment]
got it, Thanks
